cmd/minikube/cmd/delete.go

```diff
@@ -145,6 +146,39 @@ func deleteContainersAndVolumes(ctx context.Context, ociBin string) {
 	}
 }
 
+// kicbaseImage fetches images based on container label
+func kicbaseImages(ctx context.Context, ociBin string) ([]string, error) {
+	if _, err := exec.LookPath(ociBin); err != nil {
+		return nil, nil
+	}
+	delLabel := fmt.Sprintf("%s=%s", oci.CreatedByLabelKey, "true")
+	names, err := oci.ListImagesByContainerLabel(ctx, ociBin, delLabel)
+	if err != nil {
+		return nil, err
+	}
+	return names, nil
+}
+
+// printDeleteImagesCommand prints command which remove images
+func printDeleteImagesCommand(ociBin string, imageNames []string) {
+	if _, err := exec.LookPath(ociBin); err != nil {
+		return
+	}
+
+	if len(imageNames) > 0 {
+		out.Styled(style.Command, `{{.ociBin}} rmi {{.images}}`, out.V{"ociBin": ociBin, "images": strings.Join(imageNames, "")})
+	}
+}
+
+// printDeleteImageInfo prints info about removing kicbase images
+func printDeleteImageInfo(dockerImageNames, podmanImageNames []string) {
+	if len(dockerImageNames) > 0 || len(podmanImageNames) > 0 {
```

Comment:
I think it makes sense to exit early here instead if both lists are empty, otherwise the `printDeleteImagesCommand` are being run unnecessarily if we already know both image lists are empty.
```
if empty
return
// rest
```